.php_cs.dist

```diff
@@ -10,21 +20,16 @@ file that was distributed with this source code.
 HEADER;
 
 $finder = PhpCsFixer\Finder::create()
-    ->in(__DIR__)
-    ->exclude('tests/Fixtures/app/cache')
+    ->in([__DIR__.'/src', __DIR__.'/tests'])
+    ->append([__FILE__])
+    ->exclude(__DIR__.'/tests/Fixtures/app/cache')
```

Comment:
Can you revert all the changes from this line and above? But we can keep the

```php
declare(strict_types=1);
```

of course.

Reply:
I'll take care of the rest. :smile: